src/utils/types.ts

```diff
@@ -332,16 +334,21 @@ interface PageNEW {
   name: string
   progress: Progress
   reviewProgress?: ReviewProgress
+  changeRequestsProgress?: ChangeRequestsProgress
   state: PageElement[]
 }
 
 type PageElement = {
   element: ElementStateNEW
   response: ResponseFull | null
+  previousApplicationResponse: ApplicationResponse
+  latestApplicationResponse: ApplicationResponse
   thisReviewLatestResponse?: ReviewResponse
   review?: ReviewQuestionDecision
   assignmentId: number
   isAssigned?: boolean
+  isChangeRequest?: boolean
+  isChanged?: boolean
```

Comment:
Have added to structure as children to `element` in state[] on each page:
![Screen Shot 2021-03-17 at 2 59 55 PM](https://user-images.githubusercontent.com/16461988/111403197-791ca600-8731-11eb-9781-25ebfd20798d.png)

Reply:
great !